minvws/nl-kat-coordination
# Fix/report naming

### Changes

- Adds the right naming logic in the report runner.
- Introduces a first integration test for the report runner.
- Some typing fixes.

### Issue link

Closes https://github.com/minvws/nl-kat-coordination/issues/3632

### Demo

See the template and outputted names in the integration test: this is what we expect now (@Rieven right?).

### QA notes

Please go through the generate report flow and verify that the names appear as expected. Also see the issue for what we want to avoid.

---


---

## Checklist for code reviewers:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_code.md) into your comment.

---

## Checklist for QA:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_qa.md) into your comment.
